src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -82,6 +84,14 @@
     private Set<DocWorkUnit> workUnits;     // Set of all things we are going to document
 
     /**
+     * The entry point to javadoc generation. It calls {@link #startProcessDocs(RootDoc)} (String[])} from this instance.
+     * Consumers should implement this static method to call {@link #startProcessDocs(RootDoc)} from their instance.
```

Comment:
Since this method is static, it probably shouldn't mention "this instance". Can you reword this to something like "The entry point for javadoc generation. It creates an instance of HelpDoclet and calls startProcessDocs on that instance. Custom Barclay doclets should subclass this class, and implement a similar static method that creates an instance of the doclet subclass and delegates to that instance's startProcessDocs".